stage_descriptions/geospatial-05-xg4.md

```diff
@@ -0,0 +1,71 @@
+In this stage, you'll add support for responding to the `GEOPOS` command.
+
+### The `GEOPOS` command
+
+The `GEOPOS` command returns the longitude and latitude of the specified location. Its syntax is
+
+```
+GEOPOS <key> <location1>
```

Comment:
Same note as usual - avoid syntax, just show the example first. Add syntax / explanation only if it seems necessary after